src/app/components/footer/footer.component.html

```diff
@@ -55,9 +54,8 @@ <h4>Legal</h4>
       </div>
     </div>
     <div class="row justify-content-center">
-      <div class="col-3 copyright">
+      <div class="col-3 copyright text-center">
```

Comment:
```suggestion
      <div class="col-6 copyright text-center">
```
The copyright section looks like this now:
![image](https://user-images.githubusercontent.com/32552097/93198904-e97c4780-f74d-11ea-8bf5-2444587a89d3.png)

If we change `<div class="col-3 copyright text-center">` to `<div class="col-6 copyright text-center">` it would look like this.
![image](https://user-images.githubusercontent.com/32552097/93198957-fac55400-f74d-11ea-9710-596a6a65e02a.png)

I think this way it would look more consistent on all the resolutions this way.

Reply:
I see what you are going for. I totally agree this looks better, good idea! But if the screen is this small, the 'Give Feedback'-button will stand in front of the GitHub link, so you won't be able to navigate to our GitHub. How do you think about this?

Do you think the advantages of the design outweigh the disadvantages of the blocked link?

Reply:
I personally think that the button looks weird on mobile anyway, on low resolutions, it is already getting blocked (in my tests at least). 
![image](https://user-images.githubusercontent.com/32552097/93204774-ee91c480-f756-11ea-905c-9a3fc7ff28ad.png)

I think the bottom right corner is not the best location for the button on mobile in the first place, since it is the place where most people will be scrolling. 

Maybe we can move the button on a lower resolution like this:
![image](https://user-images.githubusercontent.com/32552097/93206107-2e59ab80-f759-11ea-8360-47872ea83434.png)

Maybe the button could be moved to the footer/ header instead of a sticky button and maybe add a popup when the user completed some action like adding a project or something. I'm not sure what approach is the most effective tho, I have no idea how many people give feedback etc. I think @Brend-Smits would need to decide that.

I'm not sure if we have to create a new issue to take care of the button.

Reply:
Alright! I agree with your points, thnx for the feedback

Reply:
@Brend-Smits, can we implement the button with this issue as well or should I open a new issue for it?

Reply:
New issue is recommended, looks great! Keep it up 👍🏼